### ISSUE TITLE
7.33 Nyx Mind Flare not doing expected damage

### ISSUE THREAD

**Issue by bobby-smiles** [ValveSoftware/Dota2-Gameplay#8644]
### Ability name

Mind Flare

### Description

Mind Flare should be dealing 30% of damage dealt by nyx to a target in the last 15 seconds as bonus damage. However, the ability does not behave in this way. After dealing damage to an enemy with this skill, the next cast on the same enemy within 15 seconds does not count the damage dealt by this skill into the bonus damage. In fact, the damage collected seems to reset on every cast of this ability - this does not fit with the advertised effect.

### Example Match ID (and possibly Timestamp)

General - All Matches

**Comment by Turtlestud** [comment]
I noticed that as soon as it was changed when 7.33 hit, it never showed any damage increase at all, anytime i sat in demo mode to try it out.

**Comment by AzureStriker** [comment]
It also doesn't do any damage now towards Ogre Magi, which should be one of the strongest Nukes against him considering he has 0 int growth of all the heroes in the game, but instead does exactly 0 damage, no matter what items or build on the nyx, in demo, does 0 damage to even a level 1 unbuilt, no item Ogre Magi

**Comment by 6222780106** [comment]
Agreed. From my testing, giving that everything happens within 15 seconds, it seems to account only the damage from and after the last previous Mind Flare.

**Comment by Aluerie** [comment]
#8046
